app/graphql/graphql_devise/mutations/sign_up.rb

```diff
@@ -0,0 +1,64 @@
+module GraphqlDevise
+  module Mutations
+    class SignUp < Base
+      argument :email,                 String, required: true
+      argument :name,                  String, required: false
```

Comment:
Name is an attribute only on the dummy app, we need a way to configure that, or at least remove this one for now. You can use a custom mutation in the dummy app that extends from this one